cognite/client/data_classes/files.py

```diff
@@ -46,13 +49,16 @@ def __init__(
         last_updated_time: int = None,
         cognite_client=None,
     ):
+        if labels is not None and len(labels) > 0 and not isinstance(labels[0], Label):
```

Comment:
Should the last check there go through the entire list? i.e. `and not all(isinstance(l, Label) for l in labels)`

Reply:
hmm good point, this would pass if the first element *is* a `Label` while the rest are not...

Reply:
I'm not sure how defensive it should be, or how defensive other similar parts of the code are.

Reply:
I copied this from assets, so I will need to change it there as well